validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/DepositProvider.java

```diff
@@ -152,7 +150,7 @@ public int getDepositMapSize() {
    * @param eth1DepositCount number of deposits in the merkle tree according to Eth1Data in state
    * @return
    */
-  private synchronized List<Deposit> getDepositsWithProof(
+  private List<Deposit> getDepositsWithProof(
```

Comment:
Why is there no need for synchronization here?

Reply:
It's a private method only called from methods that are already synchronised.